docs/blog/2019-02-20-getting-started-with-gatsby-themes/index.md

```diff
@@ -0,0 +1,266 @@
+---
+title: "Getting Started with Gatsby Themes and MDX"
+date: 2019-02-20
+author: Katie Fujihara
+tags: ["themes", "tutorials"]
+---
+### What is a Gatsby theme?
+> Gatsby themes allow you to focus only on the parts of the site and app building process that you need to care about by abstracting the rest away into a package. 
```

Comment:
We have a cool `<pullquote>` component you can use here!

```suggestion
<pullquote cite="Jason Lengstorf">Gatsby themes allow you to focus only on the parts of the site and app building process that you need to care about by abstracting the rest away into a package.</pullquote>
```

Reply:
@DSchau TIL, too! It would be good to document that and any other useful components. I'll create an issue for it :)